includes/REST_API/Stories_Controller.php

```diff
@@ -301,8 +301,8 @@ public function get_items( $request ) {
 
 		// Add counts for other statuses.
 		$statuses = [
-			'all'     => [ 'publish', 'draft' ],
-			'publish' => 'publish',
+			'all'     => [ 'publish', 'draft', 'future' ],
+			'publish' => [ 'publish', 'future' ],
```

Comment:
```suggestion
			'publish' => [ 'publish' ],
			'future' => [ 'future' ],
```
Would it not be cleaner to do this. At some point, we may want to filter by future stories and want a count of these?
To get future + published, simple add these together in JS.

Reply:
sure that's fine we can do that addition on the front end